web/email.ex

```diff
@@ -18,6 +18,22 @@ defmodule Community.Email do
     |> render("job_posted.text", job: job)
   end
 
+  def admin_member_added(member) do
+    new_email
+    |> to(System.get_env("ADMIN_EMAILS"))
```

Comment:
It's a comma separated list to make sure it works with bamboo. Thinks for the check.